src/test/java/Homework17.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException{
+
+        String newSongAddedNotification = "Added one song into";
+
+        openloginUrl();
+        enterEmail("vasilinapelo@gmail.com");
+        enterPassword("vasilina230109!");
+        clickSubmit();
+        searchForSong("Dark Days");
+        clickViewAll();
+        selectFirstSong();
+        clickAddTo();
+        choosePlaylist();
+        Assert.assertTrue(getNotificationMessage().contains(newSongAddedNotification));
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222884741-ad9c54d4-bb71-4fd9-8449-1ceda996b5ed.png)

Reply:
Hi Alex,
Thank you for your feedback. I've been going back and forth with this but can't find that element. Can you please help me with that?
Thanks,
Lina

Reply:
Hi dear Lina

Foe example

```
public class Homework17 extends BaseTest {

  @Test
  public void addSongToPlaylist() {
    openloginUrl();
    enterEmail("vasilinapelo@gmail.com");
    enterPassword("vasilina230109!");
    clickSubmit();
    clickAllSongs();
    chooseSong();
    clickAddTo();
    checkSongInAlbum();
  }

  public void clickAllSongs() {
    WebElement clickAllSongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
    clickAllSongs.click();

  }

  private void chooseSong() {
    WebElement song = driver.findElement(By.cssSelector(".all-songs .song-item:nth-of-type(5) .title"));
    song.click();

  }

  public void clickAddTo() {
    WebElement addToButton = driver.findElement(By.cssSelector(".btn-add-to"));
    addToButton.click();
    WebElement addToPlaylist = driver.findElement(
        By.cssSelector("section#songsWrapper .existing-playlists > ul > .playlist"));
    addToPlaylist.click();

  }

  public void checkSongInAlbum() {
    WebElement myPlaylist = driver.findElement(By.xpath("//section[@id='playlists']/ul/li[3]"));
    myPlaylist.click();
    WebElement songcheck = driver.findElement(By.cssSelector(".all-songs .song-item:nth-of-type(5) .title"));
    songcheck.isDisplayed();
  }
}
```